swagger_server/controllers_local/collectioninstrument.py

```diff
@@ -0,0 +1,300 @@
+##############################################################################
```

Comment:
Sure, when we have some standard for this / when we know what the caller wants in terms of textual response, we can standardise. Not really worth fixing until then as it will change.